### PR TITLE
Add a link to the documentation when deployment is not in RUNNING state.

### PR DESCRIPTION
**Pull request by clems4ever**
This will help users troubleshoot their deployments before asking.

### CODE REVIEW

**Comment by clems4ever** [comment]
@aboten

File: src/js/components/AppPageComponent.jsx [hunk 1]
```diff
@@ -410,6 +411,14 @@ var AppPageComponent = React.createClass({
       );
     }
 
+    var deploymentTroublesDoc = (model.status != AppStatus.RUNNING) ? (
```
Comment (oaitmous) [inline src/js/components/AppPageComponent.jsx]: This should probably be !==

File: src/js/components/AppPageComponent.jsx [hunk 2]
```diff
@@ -410,6 +411,14 @@ var AppPageComponent = React.createClass({
...
+      <div className="doc-deployment-troubles">
+        <a href="https://confluence.criteois.com/x/FcLzFw" target="_blank">
+          Having deployment troubles?
```
Comment (oaitmous) [inline src/js/components/AppPageComponent.jsx]: maybe "Having troubles deploying?"?

**Review by oaitmous — APPROVED** [review]

File: src/js/components/AppPageComponent.jsx (hunk 1 above)
Comment (clems4ever) [reply src/js/components/AppPageComponent.jsx]: ack

File: src/js/components/AppPageComponent.jsx (hunk 2 above)
Comment (clems4ever) [reply src/js/components/AppPageComponent.jsx]: will do

File: src/js/components/AppPageComponent.jsx [hunk 3]
```diff
@@ -410,6 +411,14 @@ var AppPageComponent = React.createClass({
       );
     }
 
+    var deploymentTroublesDoc = (model.status !== AppStatus.RUNNING) ? (
+      <div className="doc-deployment-troubles">
+        <a href="https://confluence.criteois.com/x/FcLzFw" target="_blank">
```
Comment (pierrecdn) [inline src/js/components/AppPageComponent.jsx:416]: FAR: make it configurable from the outside to avoid leaking these links to public repos.

**Pull request merged**